runtime/src/bank.rs

```diff
@@ -2546,14 +2546,19 @@ impl Bank {
     /// recalcuates the bank hash.
     ///
     /// Note that the account state is *not* allowed to change by rehashing.
-    /// If it does, this function will panic.
     /// If modifying accounts in ledger-tool is needed, create a new bank.
     pub fn rehash(&self) {
         let get_delta_hash = || {
-            self.rc
-                .accounts
-                .accounts_db
-                .get_accounts_delta_hash(self.slot())
+            (!self
+                .feature_set
+                .is_active(&feature_set::remove_accounts_delta_hash::id()))
+            .then(|| {
+                self.rc
+                    .accounts
+                    .accounts_db
+                    .get_accounts_delta_hash(self.slot())
+            })
+            .flatten()
```

Comment:
what am I missing in the code a little below?
Why do we call `get_delta_hash()` again?
```
let new_accounts_delta_hash = get_delta_hash().unwrap();
```

Reply:
ok